integrationTests/multiShard/relayedTx/relayedTx_test.go

```diff
@@ -33,24 +33,20 @@ type createAndSendRelayedAndUserTxFuncType = func(
 
 func TestRelayedTransactionInMultiShardEnvironmentWithNormalTx(t *testing.T) {
 	t.Run("relayed v1", testRelayedTransactionInMultiShardEnvironmentWithNormalTx(CreateAndSendRelayedAndUserTx, false))
-	t.Run("relayed v3", testRelayedTransactionInMultiShardEnvironmentWithNormalTx(CreateAndSendRelayedAndUserTxV3, true))
 }
 
 func TestRelayedTransactionInMultiShardEnvironmentWithSmartContractTX(t *testing.T) {
 	t.Run("relayed v1", testRelayedTransactionInMultiShardEnvironmentWithSmartContractTX(CreateAndSendRelayedAndUserTx, false))
```

Comment:
these calls still have a bool for isRelayedV3, will this still be needed in the future?

Reply:
updated to baseCostFixEnabled

Reply:
not truly needed, but in case we want to expand the tests

Reply:
ok